extensions/2.0/Vendor/MSFT_CollisionPrimitives/schema/glTF.MSFT_CollisionPrimitives.collider.box.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "MSFT_CollisionPrimitives glTF extension",
+    "type" : "object",
+    "description" : "Parameters describing a box collider.",
+    "allOf" : [ { "$ref" : "glTFProperty.schema.json" } ],
+    "properties" : {
+        "size" : {
+            "type" : "array",
+            "description" : "The extents of the box in each axis in local space.",
+            "items" : {
+                "type": "number",
+                "minimum": 0.0
+            },
+            "minItems" : 3,
+            "maxItems" : 3,
+            "default" : [ 1.0, 1.0, 1.0 ]
+        },
+        "extensions" : {},
+        "extras" : {}
+    },
+    "required": ["size"]
```

Comment:
Should `size` be required when it has a default? Same goes for the other collider properties.

Reply:
Good spot; my inclination is to actually remove the default values. The authoring tools which create glTF files will always some sort of concrete object with a defined size, so makes more sense to be explicit here.

Reply:
Thought about this and changed my mind here; you're right, I'll remove the defaults. Rationale is that while I don't imagine a (e.g.) 1m^3 cube isn't terribly useful, the other extensions I've looked at provide default values which at least do something and providing a default doesn't stop an application from creating a cube with a scale more representative of the scale the tool is working in.